doc/research/logo-detection/ann-benchmark.md

```diff
@@ -34,7 +34,7 @@ HNSW indexes are adjustable trhough three parameters:
 
 ## Benchmark
 
-You can find the code of the various benchmarks [here](https://github.com/openfoodfacts/openfoodfacts-ai/tree/607ec6a/logo-ann/benchmark-ann).
```

Comment:
it's better to keep a reference to a specific commit, so that if we update the code we know which version was used to generate the benchmark